pkg/eks/components/CruEKS.vue

```diff
@@ -253,91 +255,22 @@ export default defineComponent({
   computed: {
     ...mapGetters({ t: 'i18n/t' }),
 
+    needsValidation() {
```

Comment:
if I understand correctly this is used inside each validator as a guard, but wouldn't  inside the `fvExtraRules` be a better place right before returning the validation object?

We also have `hasCredential` as a computed property that returns the exact same value, can it be re-used instead?

Reply:
These are both great suggestions. I've updated the PR accordingly.